src/studio/src/designer/frontend/packages/schema-editor/src/features/editor/schemaEditorSlice.ts

```diff
@@ -187,12 +185,8 @@ const schemaEditorSlice = createSlice({
       if (state.selectedId === path) {
         state.selectedId = undefined;
       }
-      // eslint-disable-next-line no-useless-escape
-      if (path.match('[^#]\/properties')) {
-        // find parent of item to delete property.
-        const index = path.lastIndexOf('/properties/');
-        const parentPath = path.substring(0, index);
-        const propertyName = path.substring(index + 12);
+      const [parentPath, propertyName] = getParentPath(path);
+      if (parentPath !== null) {
```

Comment:
can be `if(parentPath)` unless the path sometimes is `''`

Reply:
getParentPath returns null if regex doesn't match

Reply:
Right, that will work